tutorials/python-lists-dicts/index.mdx

```diff
@@ -0,0 +1,704 @@
+
+To remove a specific item from the list, we have the `remove()` method.
+
+Remember that our list of marks currently looks like the following: `[99, 30, 68, 72, 50, 72, 83, 49, 55]`
+
+1. Use the `remove()` method to remove the mark of 68 from the list:
+
+    ```python
+    class_marks.remove(68)
+    class_marks
+    ```
+
+    Output:
+
+    ```python
+    [99, 30, 72, 50, 72, 83, 49, 55]
+    ```
+
+    <Message type="tip">
+
+      If you use the `remove()` method with an item that has more than one occurrence in the list (e.g. for the mark 72 in our case) it will remove the first occurrence of the item only.
+    
+    </Message>
+
+    Next we have the `pop()` method, which removes the item at the specified index position:
+
+2. Use the `pop()` method to remove the first mark (99, at index position 0) from the list:
+
+    ```python
+    class_marks.pop(0)
+    class_marks
+    ```
+
+    Output:
+
+    ```python
+    [30, 72, 50, 72, 83, 49, 55]
+    ```
+
+    If we want to create a new variable with the "popped" item, we can do that too:
+
+3. Use the `pop()` method to remove the third mark (50, at index position 2) and put it in a new variable:
+
+    ```python
+    single_mark = class_marks.pop(2)
+    ```
+
+4. Use the following code to check the current values of `class_marks` and `single_mark`
+
+    ```python
+    print(single_mark)
+    print(class_marks)
+    ```
+
+    Output:
+
+    ```python
+    50
+    [30, 72, 72, 83, 49, 55]
+    ```
+
+    Finally, you can use the `del` keyword to delete an item at a specified index point, or a slice of the list, or the complete list.
+
+5. Use the `del` keyword to delete the mark of 72 at index position 1:
+
+    ```python
+    del class_marks[1]
+    class_marks
+    ```
+
+    Output
+
+    ```python
+    [30, 72, 83, 49, 55]
+    ```
+
+6. Use the `del` keyword to delete the slice of the list from index positions 0 (first item) up to but not including index position 2 (third item):
+
+    ```python
+    del class_marks[0:2]
+    class_marks
+    ```
+
+    Output
+
+    ```python
+    [83, 49, 55]
+    ```
+
+    Finally, we can use the `del` keyword to delete the whole list! Indeed, we can use it to delete any type of variable we want, so that Python forgets the variable completely.
+
+7. Use the `del` keyword to delete the list:
+
+    ```python
+    del class_marks
+    class_marks
+    ```
+
+    Output:
+
+    ```python
+    NameError: name 'class_marks' is not defined
+    ```
+
+    Python no longer remembers the variable, so our request to display it after deleting it returns an error.
+    
+<Message type="tip">
+
+  Note that the syntax for appending, removing, extending etc is `list_name.method_name()`, whereas the syntax for deleting is `del list_name`. This is because append, remove, extend etc. are all **methods**, whereas del is a **keyword**. Methods and keywords have different syntax, even if they both carry out operations on a given variable or item!
```

Comment:
```suggestion
  Note that the syntax for appending, removing, extending etc. is `list_name.method_name()`, whereas the syntax for deleting is `del list_name`. This is because append, remove, extend etc. are all **methods**, whereas del is a **keyword**. Methods and keywords have different syntax, even if they both carry out operations on a given variable or item!
```